client/landing/stepper/hooks/use-create-site-hook.ts

```diff
@@ -118,7 +120,7 @@ export const useCreateSite = () => {
 	const domains = get( 'domains' );
 	const username = useSelector( getCurrentUserName );
 	const planCartItems = get( 'plans' )?.cartItems;
-	const siteTitle = get( 'newsletterSetup' )?.siteTitle as string;
```

Comment:
This seems to work. Because the `newsletterSetup` [does submit the site title](https://github.com/Automattic/wp-calypso/blob/c8160ec6534f378e8311602fe1ab3549f0c5dee3/client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx#L86). This isn't nitpick, but we want to move away from WP stores.